### ISSUE TITLE
Update notifier for non-npm install

### ISSUE THREAD

**Issue by hipstersmoothie** [intuit/auto#1313]
**Is your feature request related to a problem? Please describe.**

As a user of the `auto` binary I want know when there are updates

**Describe the solution you'd like**

`auto` tells me if there is a new update and links to the current release

**Additional context**

Might be able to use this https://www.npmjs.com/package/update-check

**Comment by mattgabor** [comment]
@hipstersmoothie when you say non-npm, do you mean this should only prompt the user to update if have installed the binary manually? I can try doing this with the package you suggested but it's geared towards globally installed npm packages.

**Comment by hipstersmoothie** [comment]
> do you mean this should only prompt the user to update if have installed the binary manually?

yeah. thinking about this more it the update warning should probably only be shown during --help or `auto info` since some of the command are expected to have specific output (like `auto version`)

> I can try doing this with the package you suggested but it's geared towards globally installed npm packages.

Might be better to roll our own inside auto then
